### PR TITLE
Embed docs in vscode-xml

### LINKED ISSUE #326: Embed docs in vscode-xml
Embed docs in vscode-xml to provide links from the settings to this embed documentation. See the following demo:

XMLEmbedDocs

### PR DESCRIPTION
**Pull request by angelozerr**
Embed docs in vscode-xml

Fixes #326

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: webview-resources/document.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+.btn svg {
+	fill: var(--vscode-button-foreground);
+}
+
+.btn:hover {
+	background-color: var(--vscode-button-hoverBackground);
+}
+
```
Comment (datho7561) [inline webview-resources/document.css:14]: Style `.btn:focus` to remove the borders from the "back to top" button after its clicked:

ButtonHighlightingMarkdownDocumentation

(to be consistent with other buttons like this in the UI such as the git initialize a repository button).

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: docs/Formatting.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
+## Formatting
```
[the remark this answers is not in the document]
Comment (datho7561) [reply docs/Formatting.md:1]: [Here are some instructions on how to do the updating in Jenkins](https://stackoverflow.com/questions/9953299/git-submodules-not-updating-in-jenkins-build)

[the remark this answers is not in the document]
Comment (angelozerr) [reply docs/Formatting.md:1]: My other idea was to update the wiki Formatting.md  to have a link to this page Formatting.md.

File: webview-resources/document.css (hunk 1 above)
Comment (angelozerr) [reply webview-resources/document.css:14]: What dif you suggested  for  `.btn:focus`  style?

Comment (datho7561) [reply webview-resources/document.css:14]: Let me try some things out quickly to see. I think something along the lines of this should work:
```css
.btn:focus {
  border: 0px;
}
```

Comment (angelozerr) [reply webview-resources/document.css:14]: I have copied/pasted this CSS from vscode-java. I suggest that you create an issue about that and when we will find a clean solution we could fix it and vscode-java too.

Comment (datho7561) [reply webview-resources/document.css:14]: Opened #330

Comment (angelozerr) [reply webview-resources/document.css:14]: Thanks @datho7561

File: docs/Formatting.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (datho7561) [reply docs/Formatting.md:1]: @fbricon What are your thoughts on this? Should we hold off on looking into this until a separate PR? Are there other options we should consider?

**Review by datho7561 — APPROVED** [review]

**Pull request merged**